### ISSUE TITLE
Linux RISC-V toolchains uses too new `libc6`

### ISSUE THREAD

**Issue by matsobdev** [raspberrypi/pico-sdk-tools#7]
Trying to try `riscv-toolchain-14-x86_64-lin.tar.gz` in Ubuntu 20.04 and:
```
libc.so.6: version `GLIBC_2.34' not found 
```
Could you build it using something older? [RISC-V](https://www.embecosm.com/resources/tool-chain-downloads/#riscv) for example provides Ubuntu 18.04 compatible compiler.

**Comment by will-v-pi** [comment]
We won't be going all the way back to Ubuntu 18.04, but can attempt building with Ubuntu 20.04 until April when it reaches end-of-life. Those toolchains you link to should have very similar performance, so I would recommend using those if you need an older supported version, or alternatively you can use the scripts in this repository to compile the toolchain from source - it should only take an hour or so, and is relatively easy on Linux systems.

**Comment by lurch** [comment]
@will-v-pi Does the existence of `riscv-toolchain-14-x86_64-lin.tar.gz` mean that the "RISC-V Toolchain for Raspberry Pi OS - for other OSs, the extension uses the compilers from Core-V" statement in `README.md` is incorrect?

**Comment by will-v-pi** [comment]
Yes, it now contains RISC-V toolchains for all supported platforms

**Comment by will-v-pi** [comment]
Closing this now, as the Ubuntu 20.04 GitHub runner is being deprecated
